vtex/utils/legacy.ts

```diff
@@ -12,6 +12,7 @@ export const toSegmentParams = (
     utmi_campaign: segment.utmi_campaign ?? undefined,
     utm_campaign: segment.utm_campaign ?? undefined,
     utm_source: segment.utm_source ?? undefined,
+    sc: segment.channel ?? '1',
```

Comment:
I think we have a `defaulSalesChannel` or something on the app. Could you check and maybe use this value?

Reply:
Isn't it from std? I couldn't find it inside the `vtex` app :/
Maybe I should leave it with `undefined` like the others?

Reply:
LGTM,
@lucis any thoughts?